Sources/PaymentsCore/Networking/HTTP.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+/// `HTTP` interfaces directly with `URLSession` to execute network requests.
+class HTTP {
+    
+    private let coreConfig: CoreConfig
+    private var urlSession: URLSessionProtocol
+    private let decoder = APIClientDecoder()
+
+    init(coreConfig: CoreConfig) {
+        self.urlSession = URLSession.shared
+        self.coreConfig = coreConfig
+    }
+
+    /// For internal use for testing purpose
+    init(urlSession: URLSessionProtocol, coreConfig: CoreConfig) {
+        self.urlSession = urlSession
+        self.coreConfig = coreConfig
+    }
+    
+    func performRequest<T: APIRequest>(endpoint: T) async throws -> (T.ResponseType) {
```

Comment:
```suggestion
    func performRequest<T: APIRequest>(request: T) async throws -> (T.ResponseType) {
```
🧶 : Makes the parameter name match the underlying generic protocol.

Reply:
This is a good call. `APIClient.fetch(endpoint: )` is used pretty heavily throughout the app. I'll make a follow-up PR addressing that for both `APIClient` and `HTTP`, so not to muddy this one.

Reply:
@sshropshire - this feedback was address in a separate PR https://github.com/paypal/iOS-SDK/pull/99